golem_messages/message/tasks.py

```diff
@@ -234,31 +234,52 @@ def __init__(self, *args, **kwargs):
 
 @library.register(TASK_MSG_BASE + 1)
 class WantToComputeTask(ConcentEnabled, base.Message):
+    """
+    A computation Offer.
+
+    A Provider sends it directly to a Requestor as a response to the Requestor's
+    Demand (Task) in order to get work (SubTask ie. TaskToCompute) to do.
+
+    Attributes:
+        node_name:          Provider's node name
+        perf_index:         Provider's Performance index
+        max_resource_size:  Provider's Storage size available for computation
+        max_memory_size:    Provider's RAM
+        price:              Offered price in GNT "WEI" (10e-18)
+        num_subtasks:       How many subtasks Provider wants to work on
+                            (simultaneously); 1 by default
+        concent_enabled:    Provider's Concent status
+        extra_data:         additional required information about the Provider's
+                            environment. `golem-messages` should be
+                            intentionally agnostic with regards to the contents
+                            of this field.
+        provider_public_key:
+                            for signing and encryption
+        provider_ethereum_address:
+                            for transactions on ETH blockchain
+        task_header:        Demand; signed by a Requestor
```

Comment:
`TaskHeader` is a class name... so unless you're also planning to update the class name, please leave the "type hint" intact here... 

```suggestion
        task_header:        TaskHeader; a Demand signed by a Requestor
```